app/styles/modal/parent-refer-teacher-modal.sass

```diff
@@ -0,0 +1,23 @@
+@import "app/styles/style-flat-variables"
+
+// This overrides the vue components modal-content style.
```

Comment:
It seems like this is being applied to a Backbone modal, not a Vue component.  Am I reading this wrong?

Also, how is this restricted to `#parent-refer-teacher-modal`?

Reply:
You are correct. The infrastructure for extending our modal template in a Vue component doesn't exist. I will revisit this by working out how to make a `ModalComponent`.